### PR TITLE
Album attributes

### PR DESCRIPTION
**Pull request by milesillsley**
No layout button yet. Visit '/photo_albums' using address bar.
PostPresenter has been modified for use with different objects.
Photo Albums have attributes: name, location and date.

### CODE REVIEW

**Review by dbugsy — APPROVED** [review]
:shipit: :shipit:

File: app/views/photo_albums/index.html.erb [hunk 1]
```diff
@@ -1,2 +1,16 @@
-<h1>PhotoAlbum#index</h1>
-<p>Find me in app/views/photo_album/index.html.erb</p>
+<h1>Albums</h1>
+
+<%= button_to 'New Photo Album', { action: 'new' }, method: :get, class: 'button' %>
+<!-- <%= link_to new_photo_album_path do%>
```
Comment (FiddlersCode) [inline app/views/photo_albums/index.html.erb]: Remove commented code

**Review by j311y — APPROVED** [review]
:shipit:

File: app/views/photo_albums/index.html.erb [hunk 2]
```diff
@@ -1,2 +1,16 @@
...
+New Photo Album
+<%end%> -->
```
Comment (FiddlersCode) [inline app/views/photo_albums/index.html.erb]: remove commented code

File: app/views/photos/new.html.erb [hunk 3]
```diff
@@ -6,5 +6,5 @@
   <%= form.file_field :image %><br>
 </p>
 
-<%= form.submit "Submit" %>
+<%= form.submit "Create album" %>
```
Comment (FiddlersCode) [inline app/views/photos/new.html.erb]: add button styling

**Review by chrisjmit — APPROVED** [review]
:shipit:

**Review by FiddlersCode — CHANGES_REQUESTED** [review]
Looks great!  Just a couple minor changes which should pop up in the relevant files.  Do you need to define any DB relationships with photos yet?

**Review by FiddlersCode — APPROVED** [review]

**Comment by FiddlersCode** [comment]
:shipit:

**Pull request merged**